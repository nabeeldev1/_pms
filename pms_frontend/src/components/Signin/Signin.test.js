import React from 'react';
// import ReactDOM from 'react-dom';
import Signin from '../Signin/Signin';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import classes from './Signin';

// describe('Component: Signin', () => {

//   it('should match its empty snapshot', () => {
//     const tree = renderer.create(
//       <Signin />
//      ).toJSON();

//     expect(tree).toMatchSnapshot();
//   });
// });

// describe('Stateless component', () => {
//     test('shallow wrapper instance should not be null', () => {
//         const wrapper = shallow(<Stateless {...props} />);
//         const instance = wrapper.instance();

//         expect(instance).not.toBe(null);
//     });
// });

describe('Statefull component', () => {
    test('shallow wrapper instance should not be null', () => {
        const wrapper = shallow(<Statefull {...props} />);
        const instance = wrapper.instance();

        expect(instance).not.toBe(null);
    });
});