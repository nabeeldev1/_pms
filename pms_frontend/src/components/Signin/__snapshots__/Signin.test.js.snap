// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Signin should match its empty snapshot 1`] = `
<div
  className={undefined}
>
  <div
    className={undefined}
  />
  <h2
    className={undefined}
  >
    Sign in
  </h2>
  <div
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <div
      className="formErrors"
    >
      
      
    </div>
  </div>
  <form
    className={undefined}
    onSubmit={[Function]}
  >
    <p>
      <input
        id="email"
        name="email"
        onChange={[Function]}
        placeholder="Email"
        type="email"
        value=""
      />
    </p>
    <p>
      <input
        id="password"
        name="password"
        onChange={[Function]}
        placeholder="Password"
        type="password"
        value=""
      />
    </p>
    <p>
      <input
        disabled={true}
        type="submit"
        value="Sign in"
      />
    </p>
  </form>
</div>
`;
